### DISCUSSION TITLE
Welcome to DeepZoom Discussions!

### DISCUSSION THREAD

**Discussion by jaybo** [jaybo/deepzoom-blog#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jamizzl** [comment]
I'm not sure if this is the right place, as there's not any discussion yet - but first, I love DeepZoom!  My question - I would love to know if there's a plan to use the current information in combination with the Routes/Trip to predict travel time (including currents)?  I love DeepZoom, and I see that it now has the route/trip feature (AMAZING) and displays the predicted current arrows onscreen for a given trip (even far into the future).  However, if I change start time by grabbing the slider, I don't see my overall route duration changinglike I would expect (Longer if my departure time means I'm fighting a tide, shorter if I get a boost).  Sorry for all the words, but is that feature already there and I'm missing it?  It would be super helpful to plan a leg of a trip (day to day) and drag the start slider to see how it changes my total travel time that day....

**Reply by jaybo** [reply]
No you're not missing this feature, and I'd love to be able to implement it, but it's just not feasible at present.  Currents are only measured at a few fixed locations.  Trying to extrapolate from these known points across a large area would be fraught with errors since hydrodynamics are complex and non-linear.  So I shift the burden onto you to estimate your speed given the displayed current velocities.

There are models which have been developed for specific areas such as [this one for the Salish Sea:](https://ecology.wa.gov/Research-Data/Data-resources/Models-spreadsheets/Modeling-the-environment/Salish-Sea-modeling)
but I'm not aware of larger scale models which could be used.

**Comment by stewput** [comment]
Hi.  I've barely started fiddling with DeepZoom since someone showed it to me this week.  Maybe it is a way to share kayaking trip reports with wind and current detail.  Through the web interface I can upload new trip *.gpx recorded on any device.  When I click on the Time readout / Calendar button at the bottom, I cannot adjust the start of an animation to anything more precise than one day.  Same with the Timebase duration drop down on the big right hand 'Trips' tab ( where also the trip length and duration do not auto fill and can't be set manually )  Can't it autodetect when a *.gpx starts and stops?  I have to manually select the start day, and manually find the start time of the trip on the 24 hour slider .  But I've just tarted trying so maybe I'm missing things.  Google browser on Linux.

**Comment by jaybo** [comment]
Try this:
1. Import the .gpx file.  At present, DeepZoom just retains the points as a **track** and ignores the timestamps.  No good reason for this omission, I just haven't done the work yet!  But for the moment...
2. Go to the **Track page** and select the track.  Click the `Track -> route` button..

![image](https://github.com/jaybo/deepzoom-blog/assets/1607323/d0172aa0-fbb0-4fea-813e-b1328356bd60)
3. Now you can manipulate the speed, and starting time of the route.

The next version may include an option dialog when importing a .gpx automatically doing step 2 on all imported tracks.

**Comment by swiftsure** [comment]
Don't know if this is the right place, but I have a question.  Is there any plans to add an overlay or layer for PNW that shows the tow lanes off the west coast?  Or can I do this myself?

**Reply by jaybo** [reply]
Here are the tow lanes updated to March 2025:

https://www.deepzoom.com/trip/vbbzth7e

One way to find things like this are to zoom in on an area where you're looking for a route, etc. and then use the search tool:

<img width="960" height="650" alt="image" src="https://github.com/user-attachments/assets/a945af63-6e91-464d-bfed-bde0c56f0907" />

**Comment by swiftsure** [comment]
That is what I was looking for.  I am a slow boat and will be offshore
overnight, so I need every advantage to avoid the crab pots, haha.  Trying
to go north from PDX, but with these diesel prices......


Have a great day,

*J. Brad Coffey*

Please insert your favorite inspirational phrase here.
Now you have been motivated, so go out and start making a difference!!!!

**Comment by swiftsure** [comment]
Looks like the updates for 2025 were only concerning the entrance to
Crescent City, CA. So no worries from PDX to Juan de Fuca


Have a great day,

*J. Brad Coffey*

Please insert your favorite inspirational phrase here.
Now you have been motivated, so go out and start making a difference!!!!
